sdk/tests/deqp/functional/gles3/es3fNegativeShaderApiTests.js

```diff
@@ -485,11 +485,11 @@ goog.scope(function() {
 
                 bufferedLogToConsole('gl.INVALID_OPERATION is generated if the size of the uniform variable declared in the shader does not match the size indicated by the glUniform command.');
                 gl.useProgram(program.getProgram());
-                gl.uniform1fv(vec4_v, data);
+                gl.uniform1fv(vec4_v, new Float32Array(1));
                 this.expectError(gl.INVALID_OPERATION);
-                gl.uniform2fv(vec4_v, data);
+                gl.uniform2fv(vec4_v, new Float32Array(2));
                 this.expectError(gl.INVALID_OPERATION);
-                gl.uniform3fv(vec4_v, data);
+                gl.uniform3fv(vec4_v, new Float32Array(3));
                 this.expectError(gl.INVALID_OPERATION);
                 gl.uniform4fv(vec4_v, data);
```

Comment:
It's better to be consistent and just get rid of data and use new Float*(4) here.